src/measure_theory/measure/content.lean

```diff
@@ -362,6 +364,56 @@ begin
   exact (μ.le_outer_measure_compacts K).trans (le_to_measure_apply _ _ _)
 end
 
+end outer_measure
+
+section regular_contents
+
+/-- A content `μ` is called regular if for every compact set `K`,
+  `μ(K) = inf {μ(K') : K ⊂ int K' ⊂ K'`. See Paul Halmos (1950), Measure Theory, §54-/
+def content_regular := ∀ ⦃K : topological_space.compacts G⦄,
+  μ K = ⨅ (K' : topological_space.compacts G) (hK: (K : set G) ⊆ interior (K' : set G) ), μ K'
```

Comment:
```suggestion
def regular := ∀ ⦃K : compacts G⦄,
  μ K = ⨅ (K' : compacts G) (hK : (K : set G) ⊆ interior (K' : set G)), μ K'
```
We're in the namespace `content` already, so the name can be shortened. Also, we have opened the namespace `topological_space`, so you can leave off that part of the name (also below).

Reply:
`content.regular` was already defined above as an instance for the statement that measures induced by contents are regular. I think it would be better to keep the names as they are to avoid renaming `content.regular` where it was already used (e.g. construction of the Haar measure).

Reply:
That is an automatically generated `instance` name. I think it would be better to rename that instance (whose name is probably not used explicitly anywhere), in order to simplify this name.